core/blockchain.go

```diff
@@ -2784,9 +2833,14 @@ func (bc *BlockChain) HandleDiffLayer(diffLayer *types.DiffLayer, pid string, fu
 		return nil
 	}
 
+	diffHash := common.Hash{}
+	if diffLayer.DiffHash.Load() != nil {
```

Comment:
`diffLayer.DiffHash.Load()` should not be nil, if it is, return error please.

Reply:
> `diffLayer.DiffHash.Load()` should not be nil, if it is, return error please.

fixed